### ISSUE TITLE
client tool does not report info message about trailing slash that online validator reports

### ISSUE THREAD

**Issue by hannob** [validator/validator#1682]
Maybe I'm just misusing the tool, but it appears to me I cannot get the vnu.jar commandline tool to provide me with the same warnings as the online tool.

Take an HTML file with an XHTML-style trailing slash tag like '<br/>':
```html
<!DOCTYPE html>
<html lang="en"><head><meta charset="UTF-8"><title>test</title></head>
<body>
Hello<br/>
World
```

The online validator (validator.w3.org) will show an info message:
```
Info: Trailing slash on void elements has no effect and interacts badly with unquoted attribute values.
```

However, I cannot get the commandline tool to report the same. It just produces no output on that file.

I checked the commandline options whether there's something needed to enable "Info" messages. The only option that seems related is --errors-only, but that says it would not report info and warning messages, and it's disabled by default. That implies to me that by default I should see the info messages.

**Comment by sideshowbarker** [comment]
Thanks for reporting this. I haven’t gone back in and looked at the code yet, but based just on my own recollection, I think I intentionally wrote that code such that it always ignores all info-level messages — and only reports errors and warnings.

The web-based checker has always emitted some info-level messages other than this (relatively recent) one for trailing-slash abuse. And my sense of things is that most of those other info-level messages aren’t useful in the command-line/automated context.

So while the simplest fix for the CLI checker not emitting the trailing-slash-abuse message would be to just make the checker start emitting all info-level messages — I don’t think that would behavior would be what most people want.

Given that, I think what needs to happen is that we special-case the trailing-slash-abuse message in some way such that by default it does get emitted by the CLI checker — and then, I guess, add a new specific CLI option to suppress it.

**Comment by sideshowbarker** [comment]
I can’t reproduce this now. It may be that I’ve changed/fixed something in the meantime, but anyway, given that it does seem to be working as expected now, I’m going ahead and closing this.
